test/unit/lib/plugins/aws/custom-resources/index.test.js

```diff
@@ -347,6 +347,48 @@ describe('#addCustomResourceToService()', () => {
       Resources.CustomDashresourceDashexistingDashs3LambdaFunction.Properties.FunctionName.length
     ).to.be.below(65);
   });
+
+  it('should configure vpc configs for the custom resource lambda function if vpc is configured', async () => {
```

Comment:
@Lokesh-Jawale all new tests should be configured with `runServerless`. see: https://github.com/serverless/serverless/tree/main/test#unit-tests

I think in tests should configure `vpc` settings for service, and then confirm in the generated template that those settings are reflected on the generated custom resource. 

(there's already one `runServerless` based test configured in context of this test file)

Reply:
Hi thanks for the review. I will update the test.